giab_runner/default.config

```diff
@@ -0,0 +1,93 @@
+# General
+[settings]
+start_nextflow_analysis = /fs2/sw/bnf-scripts/start_nextflow_analysis.pl
+
+# Run types
+[giab-single]
+assay = wgs-hg38-check
+trio = false
+case = hg002
+
+[giab-trio]
+assay = wgs-hg38-check
+trio = true
+cases = hg002,hg003,hg004
+
+[seracare]
+assay = jakob_test_oncov2-0
+trio = false
+case = seracare_case
+
+[giab-han-single]
+assay = wgs-hg38-check
+trio = false
+case = hg005
+
+# Cases
+[hg002]
+id = hg002
+clarity_pool_id = 24-236992
```

Comment:
I was guessing this isn't sensitive as it is a GIAB sample (and SeraCare below). Let me know if I'm wrong